### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by xerta555**

### CODE REVIEW

**Comment by xerta555** [comment]
Here we have `col description` which contain a space unlike all other label classnames, maybe the build fail is due to that ?

File: packages/marketplace/src/resources/locales/fr.json [hunk 1]
```diff
@@ -66,6 +66,10 @@
       "loading": "Chargement…",
       "errorLoading": "Erreur lors du chargement du README"
     },
+    "backup-restore": {
+      "col description": "Sauvegarder et restaurer toutes les données du Marché. Celà n’inclue pas les réglages ou toutes autres choses ayant été installées via le Marché."
+      "col action": "Sauvegarder / Restaurer"
+    },
```
Comment (kyrie25) [inline packages/marketplace/src/resources/locales/fr.json:72]: Please use `en.json` as a template and only change the values.

If the keys is not present there then it is not used.

**Review by kyrie25 — CHANGES_REQUESTED** [review]

**Comment by xerta555** [comment]
#CancelAndHaveAGoodDay

**Comment by theRealPadster** [comment]
@xerta555 I have opened a pull request (#369) that adds translation support for the import/export (backup/restore) modal functionality. Feel free to add the French translations once it is merged. Thanks for your contributions :)

**Comment by xerta555** [comment]
> @xerta555 I have opened a pull request (#369) that adds translation support for the import/export (backup/restore) modal functionality. Feel free to add the French translations once it is merged. Thanks for your contributions :)

My pr: https://github.com/spicetify/spicetify-marketplace/pull/370

**Pull request closed without merging**